WordPress/src/main/res/layout/person_detail_fragment.xml

```diff
@@ -1,97 +1,63 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
     SlidingRelativeLayout is used for animating the slide in-out between fragments
-    Background color is set so the fragment below it, PeopleListFragment won't be visible
+    Background color is set so the fragment below it, PeopleListFragment, won't be visible
 -->
 <org.wordpress.android.ui.SlidingRelativeLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/default_background"
-    android:orientation="vertical">
+    android:background="@color/white"
+    android:clickable="true">
 
-    <RelativeLayout
+    <include
         android:id="@+id/person_details"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+        layout="@layout/person_detail"/>
 
-        <org.wordpress.android.widgets.WPNetworkImageView
-            android:id="@+id/person_avatar"
-            android:layout_width="@dimen/avatar_sz_large"
-            android:layout_height="@dimen/avatar_sz_large"
-            android:layout_marginLeft="@dimen/margin_extra_large"
-            android:layout_marginTop="@dimen/margin_extra_large" />
+    <LinearLayout
```

Comment:
> Can I see a visual representation of your concern?

Hey @rickybanister , sure. Have a look at this screenshot of the people details screen in the current PR (annotations by me in red):
![people-mgmt-details-styling-issue](https://cloud.githubusercontent.com/assets/1032913/15268741/dea0ae02-19f1-11e6-8710-56a02ebaec1b.png)

Notice the two boxed areas. One is tappable the other is not but their styling is exactly the same (same background, text colour, text styling).

Not suggesting using a dropdown here, I've only mentioned it to point out that Calypso does differentiate between the person's username title (non tappable) and the user's role widget. In the current Android design of this feature, we mix read-only elements with read-write ones but style them the same. Perhaps we can tweak this a bit to provide more clue and make them more "separation". Lemme know if it still is not clear what i'm referring to. Cheers!